requirements-dev.txt

```diff
@@ -1,5 +1,6 @@
 -e .
-coverage
 flake8
 pytest >= 3.0.0
 pytest-django
+pytest-cov
```

Comment:
The other weird issue is that if you don't get coverage to create an XML report, codecov (maybe also coveralls does this?) reads the `.coverage` file which has a private format and creates its own XML...